src/Form/Type/OrderItemType.php

```diff
@@ -53,11 +50,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->addEventListener(FormEvents::PRE_SUBMIT, function (FormEvent $event): void {
                 $data = $event->getData();
                 if (empty($data['quantity'])) {
-                    $data['quantity'] = "1";
+                    $data['quantity'] = '1';
```

Comment:
Why is it a string?

Reply:
Who knows? xD Gonna fix it in next PR :)